docs/how-to/purge-data-on-shadow-table.rst

```diff
@@ -0,0 +1,30 @@
+==========================================
+Purge data on shadow tables
+==========================================
+
+The purge steps is disable by default and plans to run before proceeding to actual upgrade steps.
+This can be turn on with the `--purge`` flag.
```

Comment:
Some suggestions for readability.

```suggestion
The purge step is disabled by default.  When enabled, it will run before proceeding to actual upgrade steps.
This can be enabled with the ``--purge`` flag.
```